src/util/buildScript.2017071.js

```diff
@@ -296,7 +327,15 @@ export default function buildNewRxRequest(
   newRx.appendChild(buildNewRxPatient(doc, patientResource));
 
   //   Prescriber
-  newRx.appendChild(buildNewRxPrescriber(doc, practitionerResource));
+  const [prescriber, prescriberNPI] = buildNewRxPrescriber(doc, practitionerResource);
+  newRx.appendChild(prescriber);
+  if (prescriberNPI) {
+    // set the prescriber NPI in the header.from
+    xmlAddTextNodeWithAttribute(doc, header, 'From', prescriberNPI, 'Qualifier', 'C');
```

Comment:
```suggestion
    xmlAddTextNodeWithAttribute(doc, header, 'From', npi, 'Qualifier', 'C');
```

Reply:
There may be an NPI number for the pharmacy and other entities as well, naming it prescriberNPI is helpful.